packages/google_maps_flutter/google_maps_flutter/android/src/test/java/io/flutter/plugins/googlemaps/GoogleMapControllerTest.java

```diff
@@ -19,8 +20,10 @@
 import org.mockito.MockitoAnnotations;
 import org.robolectric.Robolectric;
 import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
 
 @RunWith(RobolectricTestRunner.class)
+@Config(sdk = Build.VERSION_CODES.P)
```

Comment:
This was added due to an error that complained about missing sdk 20. I was only able to get it working after looking at https://stackoverflow.com/questions/56808485/robolectric-and-android-sdk-29/57261194#57261194. I'm assuming this could also be increased depending oon what version we want to run it on